src-tauri/src/aux_functions/graph_init.rs

```diff
@@ -61,20 +62,15 @@ pub fn load_graph(data: Vec<NeighborInfo>) -> Graph {
 
 /*
 * Calculates the distance between two points on a sphere
-*
-* Conversion function:
-* Lat/Long: 1e-7 conversion from int to floating point degrees; see mesh.proto
-* Altitude: in meters above sea level, no conversion needed
 */
 fn calculate_converted_distance(x: f64, y: f64, z: f64, nbr_x: f64, nbr_y: f64, nbr_z: f64) -> f64 {
-    let conversion_factor = (10.0 as f64).powi(-7);
     return total_distance(
-        (x as f64) * conversion_factor,
-        (y as f64) * conversion_factor,
-        z as f64,
-        (nbr_x as f64) * conversion_factor,
-        (nbr_y as f64) * conversion_factor,
-        nbr_z as f64,
+        (x as f64) * LAT_CONVERSION_FACTOR,
```

Comment:
What is the purpose of the casting here? I can't see a reason this cast is necessary

Reply:
Agreed.

Reply:
Probably a holdover from something in the distant past. Removed all casts from the wrapper, thanks for the feedback! :)